app/transcription.py

```
@@ -453,36 +515,327 @@ def remove_transcription_source_JSON(self, json_file):
+            transcript.pipeline_state["stages"]["transcription"]["status"] = "pending"
+            return
+
+        try:
+            transcript.outputs["transcription_service_output_file"] = stt_files[-1]
+            # Reuse service's own finalizer to reconstruct the raw text
+            self.service.finalize_transcript(transcript)
+            self.logger.info(
+                f"[{transcript.title}] Loaded raw transcript from disk "
+                f"({os.path.basename(stt_files[-1])})."
+            )
+        except Exception as exc:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)